ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/conversions/IntFromBytes.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.conversions;
+
+import io.confluent.ksql.function.FunctionCategory;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import io.confluent.ksql.util.BytesUtils;
+import io.confluent.ksql.util.KsqlConstants;
+import io.confluent.ksql.util.KsqlException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+@UdfDescription(
+    name = "int_from_bytes",
+    category = FunctionCategory.CONVERSIONS,
+    description = "Converts a BYTES value to a INT type.",
+    author = KsqlConstants.CONFLUENT_AUTHOR
+)
+public class IntFromBytes {
+  private static final int BYTES_LENGTH = 4;
+
+  @Udf(description = "Converts a BYTES value to a INT type.")
+  public Integer intFromBytes(
+      @UdfParameter(description = "The bytes value to convert.")
+      final ByteBuffer value
+  ) {
+    return intFromBytes(value, ByteOrder.BIG_ENDIAN);
+  }
+
+  @Udf(description = "Converts a BYTES value to a INT type.")
+  public Integer intFromBytes(
+      @UdfParameter(description = "The bytes value to convert.")
+      final ByteBuffer value,
+      @UdfParameter(description = "The byte order of the number bytes representation")
+      final String byteOrder
+  ) {
+    if (byteOrder.equalsIgnoreCase(ByteOrder.BIG_ENDIAN.toString())) {
```

Comment:
We should do a null check for `byteOrder` (other two functions as well)

Reply:
Done